src/Commands/MakeTraitCommand.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Regnerisch\LaravelBeyond\Commands;
+
+use Regnerisch\LaravelBeyond\Resolvers\AppNameSchemaResolver;
+use Regnerisch\LaravelBeyond\Resolvers\DomainNameSchemaResolver;
+
+class MakeTraitCommand extends BaseCommand
+{
+    protected $signature = 'beyond:make:trait {name?} {--support} {--overwrite}';
+
+    protected $description = 'Make a new trait';
+
+    public function handle(): void
+    {
+        try {
+            $name = $this->argument('name');
+            $support = $this->option('support');
+            $overwrite = $this->option('overwrite');
+
+            $stub = $support ? 'trait.support.stub' : 'trait.stub';
+            $directory = $support ? 'Packages/Laravel/Traits' : 'Traits';
```

Comment:
`Packages/Laravel/Traits` is not longer the correct directory. Please see https://github.com/regnerisch/laravel-beyond/blob/main/src/Commands/MakeMiddlewareCommand.php#L22 as reference.